parsers/rust-taplo.zsh

```diff
@@ -0,0 +1,14 @@
+setup() {
+	cd scripts
+	cargo build --release
+}
+
+typeset -A info=(
+	lang    'Rust'
+	toml    '1.0'
+	site    'https://github.com/tamasfe/taplo/tree/master/crates/taplo'
+	src     '' # We use the Cargo.toml for now 'git@github.com:tamasfes/taplo.git'
+	version '0.12.1'
+	decoder './scripts/target/release/taplo-decoder'
+	perf    './scripts/target/release/rust-taplo-perf'
```

Comment:
I'm having a hard time getting a definitive answer from their docs, but I think Taplo also supports an encoder? If so then should add `decoder 'TODO'` so it shows up as "TODO" on the table rather than "(doesn't include encoder)".

Reply:
Or of course add a program for it, that's also fine 🙃 But don't need to.

Reply:
The problem is that you could not directly construct a `Node` but I found a workaround.

Reply:
That is likely an incompatibility between the two libraries I'm hacking together to make it so I can do encoding.

Reply:
I've not done a push since you've mentioned it.

I did look into it further.  As far as I can tell, this is not a limitation of the `taplo-encoder` implementation but of `taplo` itself.  I would either merge this as-is or have us move this to being unsupported.  I feel like TODO would be inaccurate as it implies general encoding support exists but isn't implemented yet within the matrix when in actuality, encoding support's API is extremely limited and can only be used with major caveats (performance and correctness).

Reply:
Right so; must've gotten confused with that force push 😅

I'll just merge it and mark as TODO for now. I added you to the repo so you can fix it yourself later if you so feel inclined.